pkg/kubelet/remote/remote_runtime.go

```diff
@@ -291,9 +292,13 @@ func (r *RemoteRuntimeService) ContainerStatus(containerID string) (*runtimeapi.
 // ExecSync executes a command in the container, and returns the stdout output.
 // If command exits with a non-zero exit code, an error is returned.
 func (r *RemoteRuntimeService) ExecSync(containerID string, cmd []string, timeout time.Duration) (stdout []byte, stderr []byte, err error) {
-	ctx, cancel := getContextWithTimeout(timeout)
-	if timeout == 0 {
-		// Do not set timeout when timeout is 0.
+	// Do not set timeout when timeout is 0.
+	var ctx context.Context
+	var cancel context.CancelFunc
+	if timeout != 0 {
+		// Use timeout + default timeout (2 minutes) as timeout.
+		ctx, cancel = getContextWithTimeout(r.timeout + time.Duration(timeout)*time.Second)
```

Comment:
Copied the code from `StopContainer`. Yeah, there is a mismatch. `timeout` in `StopContainer` is `int64`.

Will update this.